snuba/datasets/errors_replacer.py

```diff
@@ -227,7 +250,13 @@ def get_projects_query_flags(
         {int(group_id) for group_id in sum(results[(len(s_project_ids) + 1) :: 2], [])}
     )
 
-    return (needs_final, exclude_groups)
+    replacement_types = {
+        replacement_type.decode("utf-8")
+        for replacement_type in redis_client.mget(exclude_groups_keys_replacement_types)
```

Comment:
Should this be part of a pipeline as well? Not sure we want to make a separate request for each of these.